core/src/test/java/org/elasticsearch/index/query/SpanFirstQueryBuilderTest.java

```diff
@@ -45,22 +45,21 @@ public void testValidate() {
         int totalExpectedErrors = 0;
         SpanQueryBuilder innerSpanQueryBuilder;
         if (randomBoolean()) {
-            innerSpanQueryBuilder = new SpanTermQueryBuilder("", "test");
+            if (randomBoolean()) {
+                innerSpanQueryBuilder = new SpanTermQueryBuilder("", "test");
+            } else {
+                innerSpanQueryBuilder = null;
+            }
             totalExpectedErrors++;
         } else {
             innerSpanQueryBuilder = new SpanTermQueryBuilder("name", "value");
         }
-        SpanFirstQueryBuilder queryBuilder = new SpanFirstQueryBuilder(innerSpanQueryBuilder, 10);
+        int end = 10;
+        if (randomBoolean()) {
+            end = -1;
```

Comment:
maybe randomize this for any negative number? and add a test for the null case now that it is an Integer?